openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx

```diff
@@ -15,10 +34,53 @@ const WaitingStepList = ({ studentList, refreshData }) => {
     <Button onClick={() => refreshData()}>{gettext('Refresh')}</Button>
   );
 
+  const FindLearnerAction = (props) => {
+    const { tableInstance: { selectedFlatRows = [] } } = props || { tableInstance: {} };
+
+    const selectedFlatRowsLength = selectedFlatRows.length;
+
+    if (selectableLearners && selectedFlatRowsLength) {
+      const [row] = selectedFlatRows;
+      const {
+        values: { username },
+      } = row;
+
+      const invalidSelection = selectedFlatRowsLength !== 1;
+
+      const handleFindLearnerClick = () => {
+        if (findUsername) {
+          findUsername(username);
```

Comment:
what does it do `findUsername`? finds the username? The name is a bit misleading.